recipe/meta.yaml

```diff
@@ -150,6 +157,11 @@ outputs:
         - liblapack
         - libblas
         - nomkl
+      run:
+        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+        - cuda-cudart  # [(not win) or (cuda_compiler_version or "").startswith("12")]
```

Comment:
the reason for the selector here is non-obvious to me. Could you leave a comment please?